### ISSUE TITLE
Communication with Beach

### ISSUE THREAD

**Issue by starsuit** [FAC-Sixteen/RE4.W#8]
A place to put our daily updates

**Comment by sylv1ah** [comment]
Hi Beach!
We've been working on the wireframe for the app and getting it ready for you to use for your user testing.

The mobile link is here: https://www.figma.com/proto/IV6lJc5K9AlAhFYehxRk11c0/RE4.W?node-id=89%3A6634&scaling=scale-down

The desktop link is here: https://www.figma.com/proto/IV6lJc5K9AlAhFYehxRk11c0/RE4.W?node-id=37%3A170&scaling=min-zoom

(Tips on using Figma: click anywhere outside the frame/screen to see what is "clickable" they will be highlighted in blue and to restart from the beginning press "R" )

We are still tweaking it today and adding finishing touches so it's not entirely finished yet, but thought it would be good to show you it now for you to see. 

Bear in mind that the MVP will likely not include all of the features on display because of timing constraints, but this wireframe shows what a fully finished product could potentially look like.

For the MVP we are prioritising the recycling game itself, and being able to show the users facts and tips after each game. Authentication (login, registration) is probably going to be a stretch goal.

Today we will also be writing a user testing script/questionnaire for you to run through with your users so we can see what is or isn't working with the prototype. We will send this through later today.

**Comment by beachbagenal** [comment]
Ok great, thanks for sending through! 

Let me know when you’ve added in the final touches for today and I’ll take a look.

Is it worth organising a quick call for tomorrow to discuss next week? Not sure I’ll have results of the testing by then, but I will have taken a look at least...

Could we do 12?

**Comment by beachbagenal** [comment]
Guys just had a quick look and it looks amazing! Well done!

**Comment by sylv1ah** [comment]
Hi Beach!
 
Here's the link to the user testing questionnaire: https://docs.google.com/forms/d/e/1FAIpQLSeVDOFfEQjO5uLPah91BAvdAKzkqb4JJQY28MR1Us8ZG4ynCw/viewform

The questionnaire is going to check for what user is testing the prototype. Feel free to run through the questions and message us if there are any changes you would like us to make. 

A quick note that before every user test it would be good to go through with them that this is a prototype and it isn't a fully functioning app yet, to avoid confusion.

Please remind them that while they are having a look around that there is nothing that they can do wrong and that we just want to see how the use it and what problems they run into.

Yes we can organise a hangout session tomorrow at 12! Here's the link to that: https://hangouts.google.com/call/nXZyD9ix5HWi5Sx4A-MPAEEI?no_rd

**Comment by sylv1ah** [comment]
Here's the airtable link: https://airtable.com/invite/l?inviteId=invzPlLRoFQCSH2ad&inviteToken=8a13e15d9b93c12ce9ebdb179350c90b196879ae8359dba4220ef3e68fd64a01

**Comment by beachbagenal** [comment]
Ok cool, thanks. 

An additional bit of feedback was that it would be good to have the ability for the story to be read aloud, so kids that can’t read so well can still follow.

I’m not sure if there will be time for this, but just thought I would add it to the mix to get your thoughts. 

Cheers

**Comment by sylv1ah** [comment]
Yes that'll be a great idea to improve accessibility, we can embed voice notes for the comic which the kids can choose to play. We were thinking we could add this to our second sprint next week which is when we are planning to build the comic story. You can send us the audio you want us to use and we can play around with that.

**Comment by sylv1ah** [comment]
Tuesday 18th afternoon: we will send you the app to test
Wednesday 19th: 10:30am: scheduled call

**Comment by sylv1ah** [comment]
Also here are some other issues that may need amending with the Airtable data:
* the Recycle field should only strictly be a yes or a no in order for us to feed this rule to the recycling and trash bins (e.g for the plastic toys it says "well sort of..." and for the sandwich packets it says yes but followed with a tip)
* we also found the duplicate image, for colgate toothpaste the image is a post it note 
![image](https://user-images.githubusercontent.com/35779506/59508418-c5c0b680-8ea5-11e9-8689-287d3fecab37.png)

**Comment by beachbagenal** [comment]
Ok cool, let me take a look this weekend and have it with you by Monday morning.

Do I have access to the air table?

**Comment by sylv1ah** [comment]
Yes you should have access from this link https://airtable.com/invite/l?inviteId=invzPlLRoFQCSH2ad&inviteToken=8a13e15d9b93c12ce9ebdb179350c90b196879ae8359dba4220ef3e68fd64a01

**Comment by beachbagenal** [comment]
Hey guys, you should see the table updated with information for all the majority of fields now.

Let me know if you have any questions!

**Comment by beachbagenal** [comment]
Hey did we say we were going to have a call tomorrow? I have it in my diary for today but think we said you send over a version tonight and then go through it on a call tomorrow?

**Comment by sylv1ah** [comment]
Hey Beach, Yes we scheduled a call for tomorrow after we send you what we have so far , a few things have held us back a bit such as currently the game is only working on desktop only. But we are aiming to send you something tonight.

Also, for the product handover on Friday this week, we were wondering what time we could schedule that it's usually scheduled between 10-12

**Comment by sylv1ah** [comment]
note: the duracel battery image link is broken

**Comment by beachbagenal** [comment]
Hi Sylvia, OK cool. Well looking forward to seeing it tonight if possible. For the product handover, I was meaning to speak to Charlie about this. I am not going to be available on the Friday unfortunately, but am on the Monday and was wondering if we could do it then? I will send him a note now, and let you know what he comes back with. 

So for the images, is it the duracell battery and the post it that need replacing?

Cheers

**Comment by sylv1ah** [comment]
Update: we may have a solution for mobile! And here's the current project so far: https://recycling-superheroes.netlify.com/ bear in mind it is looking a bit clunky right now but you should hopefully see that the game is working. Tomorrow we will work on polishing the pages!

Also if you are not available to come to us on the Friday is it possible to do the handover remotely? via video call? it would probably start around 2pm (on Monday we have a whole day workshop so we wouldn't have time to schedule a handover, you can try emailing Charlie to sort something out if Friday is still an issue)

**Comment by sylv1ah** [comment]
Hey Beach, tomorrow is scheduled for user testing and was wondering if you have/can get anything prepared, much like testing with the prototype but now with the actual product instead. So far we have much of the styling done if you check the link again. Will update at the end of the day

**Comment by beachbagenal** [comment]
Hey yes that’s is fine. When is it available to test? Will you be making updates tomorrow if we can provide feedback tonight?

**Comment by sylv1ah** [comment]
Hey Beach, it's all ready and you are able to use it to test with. The only thing is that it isn't 100% on mobile as right now we were able to get the drag to work but can't drop it into any bins. So for testing you should try to only use it on a computer. Same thing with the prototyping when you gather the feedback to upload it as another issue.

Today is our last build day and we have been fixing minor bugs and also added a credits page to credit the icons used. Tomorrow we are scheduled to do product handovers and presentations to the rest of our cohort, was wondering if you were able to discuss the times of the product handover with Charlie?

**Comment by beachbagenal** [comment]
Ok cool, will review it tonight. What do you you mean when you say ‘same thing with the prototyping when you gather the feedback to upload it as another issue’? 

Ok understood on the mobile issue, I take this isn’t something that you’ll be able to fix today? 

Do you have any idea how big an issue this would be to fix? 

Yes, think we’re going to do it at 2 tomorrow. Looking forward to it!

**Comment by sylv1ah** [comment]
Ah ok so like how you gave us feed back from prototype testing (here: #15 ), even though we may not have time to amend the issues that users will find it would be a good insight for the next steps on where you want to take the app after we hand it over to you. 

With the mobile issue, it is definitely something we'd have to spend more time to fix (we wouldn't have time to have it ready for your testing), since this is a web application that we are building there are certain things thats limited to what you can do on both desktop and mobiles. There are ways but if the direction you want to take your product heavily depends on having mobile usability then it would be an idea to create a native mobile app as well alongside the desktop version.

**Comment by sylv1ah** [comment]
**Update!**
Sam has managed to fix the mobile issue so you can actually use it to test with. 🎉

also minor side note for Airtable: some images have white backgrounds on them so you may see a few with them in the game. If/when you want to amend or add to the table, it might be worth choosing images with transparent backgrounds

**Comment by beachbagenal** [comment]
OK great! 

However when I click on the link i just get a Not Found white page...Are you able to update so we can test it?

Cheers

**Comment by starsuit** [comment]
Hi Beach!

Try this one: https://recycling-heroes.netlify.com/ 
I think we renamed it to 'heroes' from 'superheroes' at some point

RE the touchscreen: it works but it's a teensy bit clunky so again you might want to look into a proper (app store etc) mobile app at some point in the future. None of us have much experience with mobile development but we've done what we can!

Sam
